corehq/form_processor/models.py

```diff
@@ -722,6 +722,23 @@ def get_obj_by_id(cls, case_id):
         from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
         return CaseAccessorSQL.get_case(case_id)
 
+    @property
+    @memoized
+    def parent(self):
+        """
+        Returns the parent case if one exists, else None.
+        NOTE: This property should only return the first parent in the list
+        of indices. If for some reason your use case creates more than one,
+        please write/use a different property.
```

Comment:
ok, i had just implemented the sql version of https://github.com/dimagi/commcare-hq/blob/auto-case-close-reference-parent/corehq/ex-submodules/casexml/apps/case/models.py#L189 , but i can update both.  is it true that this identifier is always 'parent' for all cases created by CommCare or should i instead be ignoring this index `identifier` field altogether and only relying on the relationship being 'child': 

https://github.com/dimagi/commcare-hq/blob/auto-case-close-reference-parent/corehq/ex-submodules/casexml/apps/case/sharedmodels.py#L24
https://github.com/dimagi/commcare-hq/blob/auto-case-close-reference-parent/corehq/form_processor/models.py#L866